numpy/core/overrides.py

```diff
@@ -152,6 +152,8 @@ def public_api(*args, **kwargs):
         if module is not None:
             public_api.__module__ = module
 
+        public_api.__skip_array_function__ = implementation
+
         return public_api
```

Comment:
This will not merge cleanly after #13529

Reply:
fixed